### PR TITLE
[Minor] improve(doc) : Update Paimon Catalog doc in `manage-relational-metadata-using-gravitino.md`

### PR DESCRIPTION
**Pull request by caican00**
### What changes were proposed in this pull request?

Add Paimon catalog information to the `manage-relational-metadata-using-gravitino.md` doc.

### CODE REVIEW

**Comment by caican00** [comment]
cc @FANNG1

File: docs/manage-relational-metadata-using-gravitino.md [hunk 1]
```diff
@@ -91,6 +92,7 @@ Currently, Gravitino supports the following catalog providers:
 |---------------------|--------------------------------------------------------------------------------|
 | `hive`              | [Hive catalog property](./apache-hive-catalog.md#catalog-properties)           |
 | `lakehouse-iceberg` | [Iceberg catalog property](./lakehouse-iceberg-catalog.md#catalog-properties)  |
+| `lakehouse-paimon`  | [Paimon catalog property](./lakehouse-paimon-catalog.md#catalog-properties)   |
```
Comment (FANNG1) [inline docs/manage-relational-metadata-using-gravitino.md:95]: could you format the table?

**Comment by FANNG1** [comment]
LGTM, except minor comments

File: docs/manage-relational-metadata-using-gravitino.md (hunk 1 above)
Comment (caican00) [reply docs/manage-relational-metadata-using-gravitino.md]: updated.

**Comment by caican00** [comment]
> LGTM, except minor comments

updated, cc @FANNG1

**Pull request merged**